go.mod

```diff
@@ -7,59 +7,59 @@ require (
 	github.com/Nvveen/Gotty v0.0.0-20120604004816-cd527374f1e5 // indirect
 	github.com/containers/buildah v1.20.1
 	github.com/containers/common v0.35.4
-	github.com/containers/image/v5 v5.10.5
+	github.com/containers/image/v5 v5.10.6
 	github.com/containers/storage v1.29.0
 	github.com/docker/distribution v2.7.1+incompatible
 	github.com/elazarl/goproxy v0.0.0-20191011121108-aa519ddbe484 // indirect
 	github.com/fsouza/go-dockerclient v1.7.2
 	github.com/opencontainers/runc v1.0.0-rc93
 	github.com/opencontainers/runtime-spec v1.0.3-0.20200929063507-e6143ca7d51d
-	github.com/openshift/api v0.0.0-20210222200543-70b287c0a0a5
-	github.com/openshift/client-go v0.0.0-20201020074620-f8fd44879f7c
+	github.com/openshift/api v0.0.0-20210331193751-3acddb19d360
+	github.com/openshift/client-go v0.0.0-20210331195552-cf6c2669e01f
 	github.com/openshift/imagebuilder v1.2.0
-	github.com/openshift/library-go v0.0.0-20201123212217-43f358922ea0
+	github.com/openshift/library-go v0.0.0-20210430084706-e555322cb708
 	github.com/openshift/source-to-image v1.3.1
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/cobra v1.1.3
 	github.com/spf13/pflag v1.0.5
 	golang.org/x/sys v0.0.0-20210324051608-47abb6519492
-	k8s.io/api v0.20.0
-	k8s.io/apimachinery v0.20.0
-	k8s.io/client-go v0.20.0
-	k8s.io/component-base v0.20.0
-	k8s.io/klog/v2 v2.4.0
-	k8s.io/kubectl v0.20.0
-	k8s.io/kubernetes v1.20.0
+	k8s.io/api v0.21.0-rc.0
+	k8s.io/apimachinery v0.21.0-rc.0
+	k8s.io/client-go v0.21.0-rc.0
+	k8s.io/component-base v0.21.0-rc.0
+	k8s.io/klog/v2 v2.8.0
```

Comment:
of course klog is separate enough that this is fine

Reply:
This probably got auto-bumped with the k8s update.

Reply:
yeah, since newer `k8s` brings new `klog`, it got bumped here, too. if we need to keep it down, can add `replace`, but i don't think it's really necessary.